v2.0/start-a-local-cluster.md

```diff
@@ -181,6 +181,10 @@ As mentioned earlier, CockroachDB automatically replicates your data behind-the-
 
 The replica count on each node is identical, indicating that all data in the cluster was replicated 3 times (the default).
 
+{{site.data.alerts.callout_info}}
```

Comment:
Same as for v1.1 version:

To keep these getting started pages lean, perhaps instead of embedding the include, we could link to the known limitation, for example:

```
{{site.data.alerts.callout_info}}Capacity metrics can be incorrect when running multiple nodes on a single machine. For more details, see this [limitation](known-limitations.html#available-capacity-metric-in-the-admin-ui).{{site.data.alerts.end}}
```

Reply:
Addressed in eacc8ede